OpenRA.Mods.Common/Activities/Attack.cs

```diff
@@ -64,6 +64,14 @@ protected virtual Activity InnerTick(Actor self, AttackBase attack)
 			if (!attack.Info.IgnoresVisibility && type == TargetType.Actor && !Target.Actor.HasTrait<FrozenUnderFog>() && !self.Owner.CanTargetActor(Target.Actor))
 				return NextActivity;
 
+			// Drop the target if it is being captured by our own team.
+			// Cannot check for sabotage-in-progress without reference inside capturable object to the object doing the capturing.
+			var capturable = Target.Actor.TraitOrDefault<Capturable>();
```

Comment:
Looking at how `Capturable` works, `BeingCaptured` is true for only about half a tick, so you might as well skip that (the capture is instant and the check doesn't give you anything). The one that you want is `ExternalCapturable`, which takes a lot longer.